tools/db/build/common.php

```diff
@@ -21,4 +21,25 @@ function cache($url, $filename, $duration = 60 * 60 * 24 * 7, $do_force = false)
     }
     return true;
   }  
+
+  /*
+   Recursively delete files from a folder plus the folder itself
+  */
+  function deleteDir($dirPath) {
+    if (!is_dir($dirPath)) {
+      throw new InvalidArgumentException("$dirPath must be a directory");
+    }
+    if (substr($dirPath, strlen($dirPath) - 1, 1) != '/') {
+      $dirPath .= '/';
+    }
+    $files = glob($dirPath . '*', GLOB_MARK);
```

Comment:
Some posters on [SO](https://stackoverflow.com/questions/3349753/delete-directory-with-files-in-it) mention `glob` doesn't pick up hidden files, so they suggestion`scandir`.

Reply:
True. In this use-case it won't matter because we know which files are placed in that folder, because they are unzipped from a file we create anyway. So I think I'm comfortable to leave it as is :)